src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.sb__form {
+  margin: 20px 0;
+}
+
+.sb__form__small {
+  width: 50%;
+}
+
+.search__bar {
+  width: 100%;
+
+  color: #3d4e61;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  outline: none;
+}
+
+.search__bar-big {
+  padding: 25px 62px 23px;
+
+  font-size: 16px;
+  line-height: 22px;
+
+  background: url(images/Search.svg) no-repeat 26px 25px;
+}
+
+.search__bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__bar-big:focus {
+  font-weight: bold;
+  box-shadow: none;
+
+  background: url(images/Search.svg) no-repeat 26px 25px,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search__bar-small {
+  padding: 12px 33px 10px;
+
+  font-size: 14px;
+  line-height: 19px;
+  
+  text-shadow: 0 3px 2px rgba(0, 0, 0, 0.25);
+  background: url(images/Search.svg) no-repeat 13px 15px;
+  background-size: 11px 11px;
+}
+
+.search__bar-small:hover {
+  text-shadow: none;
+}
+
+.search__bar-small:focus {
+  font-weight: bold;
+  text-shadow: none;
+
+  background: url(images/Search.svg) no-repeat 13px 15px,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-size: 11px 11px, 100% 100%;
+
+  box-shadow: none;
+  outline: none;
+}
```

Comment:
`search__bar-small:focus` and `search__bar-big:focus` has the same styles. Move it to the new common class. And use this class

Reply:
I tried my best at cutting on repetitive properties. I got rid of a lot of them (look at my commit 'version 3.0'. I also tried dealing with background, and I partly succeeded. Anyway, when I do the thing the used ask here, the needed behavior crashes.